variant_remapping_tools/tests/test_reverse_strand.py

```diff
@@ -0,0 +1,81 @@
+from collections import Counter
+from unittest import TestCase
+from unittest.mock import Mock
+
+import pysam
+from pysam.libcalignedsegment import AlignedSegment
```

Comment:
This does not look like it is being used...

Reply:
removed